### PR TITLE
Add crt to ARO Docker container for AME Root

### PR DESCRIPTION
**Pull request by cloudfit-nick**
Add AME Root the ARO RP Docker Container

The AppLens endpoint is using an AME Root cert so the CA needs to be added to the container so that Go will not reject the endpoint.

ARO-2612

### CODE REVIEW

File: Dockerfile.aro [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ARG REGISTRY
 FROM ${REGISTRY}/ubi8/ubi-minimal
 RUN microdnf update && microdnf clean all
+RUN curl -o /etc/pki/ca-trust/source/anchors/AMEROOT_ameroot.crt http://crl.microsoft.com/pkiinfra/certs/AMEROOT_ameroot.crt && update-ca-trust
```
Comment (petrkotas) [inline Dockerfile.aro:4]: How do we know this is the correct certificate? I see it is over http and it is public. Shoud it be?

Comment (cloudfit-nick) [reply Dockerfile.aro:4]: That is a good question, I also was surprised it was not over https.  I will look into it more.

Comment (weinong) [reply Dockerfile.aro]: the thumbprint is verified in Readme.md

**Review by weinong — APPROVED** [review]
LGTM

**Review by bennerv — COMMENTED** [review]
Is there a well-known endpoint for this certificate?  Can we not just issue a request to the certificate on run, and then modify our tls config to trust the additional CA?  

We do something similar with client certificates for the ARM client authorizer.  `pkg/util/clientauthorizer/arm.go`

Then when the CA expires, assuming they update the well-known endpoint we will not have to make any code changes.

**Comment by weinong** [comment]
@bennerv indeed. https://aka.ms/getissuers can be leveraged to get this CA, I think

**Comment by cblecker** [comment]
Closing in favour of #2838

**Pull request closed without merging**